### ISSUE TITLE
Additional support to set up a fresh environment 

### ISSUE THREAD

**Issue by mhodki** [uwasystemhealth/shl-mkdocs-tutorial-and-template#2]
On my computer I did the following:

```bash
conda create -n mkdocstutorial python
conda activate mkdocstutorial
```
then enter the 
```bash
pip install -r requirements.txt
```

**Comment by frinzekt** [comment]
Thanks for letting us know about this Melinda! Would you like to send us a pull request to add the information in the documentation?

This would be a great candidate to be an optional admonition inside the admonition for installation of python. To create it as an optional admonition, omit the "+" when doing it.

Eg. 

```
??? info "Set up in a virtual environment"
       ... Your content here ...
```

To send a pull request, you could create a new branch from `main`. Commit+Push the change there, and go to "GitHub" pull request and there should be a shiny green button "Create a pull request" for your branch back to main. :)

**Comment by frinzekt** [comment]
This is now addressed by #3
